bootstrap/crds/crds-wc.txt

```diff
@@ -20,3 +20,21 @@ velero/serverstatusrequests.yaml
 velero/volumesnapshotlocations.yaml
 gatekeeper/gatekeeper.yaml
 gatekeeper/gatekeeper-constraints.yaml
+ambassador/filter.yaml
+ambassador/filterpolicy.yaml
+ambassador/authservices.yaml
+ambassador/devportals.yaml
+ambassador/hosts.yaml
+ambassador/kubernetesendpointresolvers.yaml
+ambassador/kubernetesserviceresolvers.yaml
+ambassador/logservices.yaml
+ambassador/mappings.yaml
+ambassador/modules.yaml
+ambassador/ratelimitservices.yaml
+ambassador/tcpmappings.yaml
+ambassador/tlscontexts.yaml
+ambassador/tracingservices.yaml
+ambassador/project.yaml
+ambassador/projectcontroller.yaml
+ambassador/projectrevision.yaml
+ambassador/ratelimit.yaml
```

Comment:
I wrote about it in the description. Im not sure if I should extend it to SC since the task was only for wc? I did add the config option to sc but when i realized i needed to change in all ingresses as well i went back to only wc.

Reply:
Aah, I missed that part. But if we are suppose to support it, we should support it for both clusters? Or do we have a usecase for only supporting it in wc?

Reply:
This issue is scoped only to allowing ck8s end users (wc users) to use Ambassador as ingress (as alternative to nginx). If we want some Ambassador features also in sc is another discussion and issue.

Reply:
When we are becoming a OS solution provider, can we think of the sc owner as us? Aren't we also just a customer that uses the project?
I'm just thinking that if a customer wants to run a cluster with ambassador, is it a cluster we manage then? If we are to manage it, can't we decide on what ingress are being used. And if it's a cluster that somebody else is going to manage and they want ambassador on one cluster. I think the chances are big that they want ambassador on the other cluster as well.

Reply:
But I guess it doesn't matter if we split it into two steps. But I see little point in doing it since the addition is small to add it in this PR as well

Reply:
I agree with you Fredrik. Not sure i will have the time to fix that as well though. It still requires changing all the charts using ingresses to add an annotation. And testing it of course.
What do you think @tordsson